blueman/main/Manager.py

```diff
@@ -72,17 +71,16 @@ def do_activate(self) -> None:
             # Connect to configure event to store new window position and size
             self.window.connect("configure-event", self._on_configure)
 
-            grid = self.builder.get_widget("grid", Gtk.Grid)
-
             toolbar = self.builder.get_widget("toolbar", Gtk.Toolbar)
             statusbar = self.builder.get_widget("statusbar", Gtk.Box)
 
+            self._infobar = self.builder.get_widget("message_area", Gtk.InfoBar)
+            self._infobar.connect("response", self._infobar_response)
+            self._infobar_bt: str = "empty"
```

Comment:
Shouldn't this just be an empty string?

Reply:
Yeah, leftover from testing. I'll fix that.

Reply:
Fixed.